src/main/java/com/dj/core/model/graph/ConnectedNeuron.java

```diff
@@ -225,6 +234,10 @@ private boolean forwardCalculated() {
         return signalReceived == 0;
     }
 
+    private int neuronIndex(Neuron n) {
```

Comment:
please make input argument final, just to be consistent with other places (or remove final in other places :) )

Reply:
Done